common/src/main/resources/conf/production.conf.json

```diff
@@ -0,0 +1,4 @@
+{
+  "odbURI": "wss://lucuma-odb-production.herokuapp.com/ws",
```

Comment:
Should we use the `gemini.edu` domain?

Reply:
And the `lucuma.xyz` for staging... I'll look into setting these up.